source/graphql/authenticate.txt

```diff
@@ -15,130 +13,127 @@ Authenticate GraphQL Requests
 Overview
 --------
 
-The Atlas GraphQL API operates over HTTP, which means that you can access your
-:ref:`exposed data <graphql-expose-data>` using any HTTP or GraphQL client.
-Atlas App Services enforces rules for all GraphQL operations, so any GraphQL HTTP request
-must include an application user's login credentials or a valid access token.
+The GraphQL API requires that incoming requests include authentication
+information for the user making the request. This lets the API enforce
+rules and validate document schemas for each operation.
+
+Users authenticate requests by providing authentication information in
+the request headers. They can either use an authentication access token
+or directly include login credentials from any of the following
+authentication providers:
 
-.. _graphql-authorization-token-header:
+- :ref:`Email/Password <email-password-authentication>`
 
-Authorization Header
---------------------
+- :ref:`API Key <api-key-authentication>`
 
-To authenticate a GraphQL request as an arbitrary logged in user from any
-provider, include a valid :ref:`user access token
-<authenticate-http-client-requests>` as a bearer token in the request's
-``Authorization`` header:
+- :ref:`Custom JWT <custom-jwt-authentication>`
 
-.. code-block:: javascript
-   :emphasize-lines: 4
+.. note::
    
-   http.post({
-     "url": "https://realm.mongodb.com/api/client/v2.0/app/<yourappid-abcde>/graphql",
-     "headers": {
-       "Authorization": "Bearer <Access Token>"
-     },
-     "body": '{"query":"query AllMovies {\n  movies {\n    title\n    year\n  }\n}"}'
-   })
+   You must :ref:`enable an authentication provider
+   <authentication-providers>` before users can authenticate with
+   it.
```

Comment:
ditto question in https://github.com/mongodb/docs-app-services/pull/216/files#r1033975352